Sources/BraintreeCore/Public/BraintreeCore/BTWebAuthenticationSession.h

```diff
@@ -0,0 +1,14 @@
+
+#import <Foundation/Foundation.h>
+
+#import <AuthenticationServices/AuthenticationServices.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BTWebAuthenticationSession : NSObject
+@property (nonatomic, weak) id<ASWebAuthenticationPresentationContextProviding> presentationContextProvider API_AVAILABLE(ios(13));
+
+- (void)startWithURL:(NSURL*)url callbackURLScheme:(NSString *)callbackURLScheme completionHandler:(void (^)(NSURL * _Nullable callbackURL, NSError * _Nullable error))completionHandler;
+@end
```

Comment:
I wanted to make this class internal but I was having some trouble with headers and compilation.